x-pack/legacy/plugins/code/server/__tests__/multi_node.ts

```diff
@@ -63,7 +63,8 @@ const xpackOption = {
   },
 };
 
-describe('code in multiple nodes', () => {
+// Skip because of issue: https://github.com/elastic/code/issues/1387
```

Comment:
@spacedragon there are still some errors for this test. skip now. let's have a follow up fix to enable this.